mappings/net/minecraft/entity/passive/RabbitEntity.mapping

```diff
@@ -39,10 +50,15 @@ CLASS net/minecraft/class_1463 net/minecraft/entity/passive/RabbitEntity
 			ARG 1 type
 	CLASS class_1467 RabbitJumpControl
 		FIELD field_6855 rabbit Lnet/minecraft/class_1463;
+		FIELD field_6856 inJump Z
```

Comment:
`jumping`?

Reply:
this is like in air... this is really just set, not read in any way.